Riot/Modules/SecureBackup/Setup/SecureBackupSetupCoordinator.swift

```diff
@@ -57,6 +57,8 @@ final class SecureBackupSetupCoordinator: SecureBackupSetupCoordinatorType {
     init(session: MXSession, checkKeyBackup: Bool = true, allowOverwrite: Bool = false, navigationRouter: NavigationRouterType? = nil, cancellable: Bool) {
         self.session = session
         self.recoveryService = session.crypto.recoveryService
+        
+        session.vc_homeserverConfiguration()
```

Comment:
What does this call do?

Reply:
Oupsie, nothing 😓 I saw it, said I was gonna remove it, never did 🤦